onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -4811,10 +4811,11 @@ def aten_rad2deg(self: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_rand(size: INT64) -> TensorType:
+@torch_op("aten::rand")
+def aten_rand(size: Sequence[int]) -> TReal:
```

Comment:
Size is symint. We may need to update the spec to promote size to Input. Cc @gramalingam